web/assets/css/_vars.less

```diff
@@ -1,11 +1,11 @@
-@base00: #1e1d1d;
-@base01: #2a2929;
-@base02: #3d3c3c;
-@base03: #504b4b;
-@base04: #868585;
-@base05: #e6e7e8;
-@base06: #f5f5f5;
-@base07: #ecf0f1; /* light grey (pending) */
+@base00: #151515;
+@base01: #262626;
+@base02: #363636;
+@base03: #4D4D4D;
+@base04: #808080;
+@base05: #E2E2E2;
+@base06: #F2F2F2;
+@base07: #E2E2E2; /* light grey (pending) */
```

Comment:
Noticed these @base## colours were hanging around; is this just because we're at the midpoint and these are being used elsewhere? If so, NP, maybe we can clean them up with our next iteration.

Reply:
good call, i can review these and see if they can be removed / changed to the colours below!

Reply:
i think removing all the base variables from the less section is done, just wanted to see if wats passed